### DISCUSSION TITLE
Is it possible to hide other elements based on Link Authentication status ?

### DISCUSSION THREAD

**Discussion by AnatoleT** [joshnuss/svelte-stripe#118] [Q&A] [answered]
Hello world, 
I implemented the link authentication component, and I was wondering if that's possible to hide other components (like Card Number, Card CVC, Card Expiry and Express Checkout) if the user has a Link account. 
(with the user email in default values of LinkAuthentication component).

Happy coding :)

**Comment by joshnuss** [comment]
Hi @AnatoleT,

I believe it should work that way out of the box.

If you specify the customer's email as a default value, it will collapse all the fields in Payment element

Is it not working?
Maybe you have to allow Link when creating the payment intent

**Reply by AnatoleT** [reply]
Indeed it is. 
However I would like to hide (custom) elements based on the status of the user (connected or not with Link). Is it possible ? :)

**Reply by joshnuss** [reply]
That's a good question, I'm not sure.

Maybe there is an event you can hook into. I would ask in the Stripe Discord channel, as I think it's a general question about Stripe Elements.

Feel free to update this discussion if you find a solution

**Comment by AnatoleT** [comment, marked as answer]
Update : I contacted the stripe support on their Discord. It seems there is no possibility for now to hide elements based on the Link Authentication state.
